src/services/tripServieces.js

```diff
@@ -0,0 +1,5 @@
+import database from '../models';
```

Comment:
This is not a database but an encapsulation of all the models.

Reply:
I'm gonna rename it